modules/op-release-notes-1-19.adoc

```diff
@@ -246,7 +246,7 @@ spec:
 
 * With this update, the Git resolver included in the remote resolution feature now uses the native git binary instead of the pure Go `go-git` library. This change reduces memory consumption and improves clone performance, especially for large repositories. This enhancement uses shallow-clone flags, for example `--depth 1`, to reduce resource usage. No changes to pipeline manifests are required.
 
-* With this update, the `onError` field in {pipelines-title} supports Tekton parameter substitution. Previously, the `onError` filed only accepted literal values, such as `continue`, `stop`, or `finally`. You can use substitution tokens, such as `$(params.strategy)`, to dynamically determine failure handling behavior at runtime. This allows a single Pipeline definition to adapt its `onError` policy based on parameters, context, or results.
+* With this update, the `onError` field in {pipelines-title} supports Tekton parameter substitution. Previously, the `onError` field only accepted the literal values `stopAndFail` and `continue`. You can now use substitution tokens, such as `$(params.strategy)`, to dynamically determine failure handling behavior at runtime. This allows a single Pipeline definition to adapt its `onError` policy based on parameters, context, or results.
```

Comment:
Consider changing this sentence to the feedback provided in this [comment](https://github.com/openshift/openshift-docs/pull/96543/files#r2236611503).